### ISSUE TITLE
Add code to Perform 2's Complement of Binary Number 

### ISSUE THREAD

**Issue by kunalshah017** [kunalshah017/Play-With-Binaries#3]

**Comment by dishamodi0910** [comment]
It will be better, if we will do this like take 1's complement and add 1 to it. Like first toggling all the bits and then adding 1 to it. And the resultant will be 2's complement.

**Comment by kunalshah017** [comment]
> It will be better, if we will do this like take 1's complement and add 1 to it. Like first toggling all the bits and then adding 1 to it. And the resultant will be 2's complement.

yes you are correct, gonna do it that way, as we already have a function to give 1's complement we will use that and make logic that can do binary addition for adding that 1.

thanks for the suggestion, I have not yet made any code to do binary addition if you have any code for that please do fork it into the repo 

Thank You 😄

**Comment by dishamodi0910** [comment]
I have added the function for 2's complement will be adding for addition of 2 numbers as well today itself.

**Comment by dishamodi0910** [comment]
Instead of using integer array, I believe using vectors would be more easy as it's easy to pass the reference in that case, so we can keep our changes preserved.
Some suggestions from my side for more operations regarding binaries as the repo itself says that playing with binaries: So we can include things like :
1)Check if ith bit is set or not.
2)If ith bit is set, clear it.
3)Check number of set bits in a number
4)Find from binary number whether that number is odd or even in decimal.
5)Check if a given binary number is power of 2,4 or 8.
6)Subtract 2 binary numbers would be interesting as the borrow logic would be amazing to be programmed.
